scripts/build.js

```diff
@@ -53,7 +53,7 @@ function handler(config, err, stats) {
 console.log('Compiling relay queries...');
 const relayCompiler = spawn(
   path.resolve('./node_modules/.bin/relay-compiler'),
-  relayCompilerArguments,
+  relayCompilerArguments.concat('--watchman', 'false'),
```

Comment:
relay updated this to be on by default which throws an error when you just try to build normally